src/NuGet.Clients/PackageManagement.UI/Xamls/InfiniteScrollList.xaml.cs

```diff
@@ -87,10 +89,15 @@ public bool CheckBoxesEnabled
         public PackageItemListViewModel SelectedPackageItem => _list.SelectedItem as PackageItemListViewModel;
 
         // Load items using the specified loader
-        internal void LoadItems(IItemLoader<PackageItemListViewModel> loader, string loadingMessage, INuGetUILogger logger)
+        internal void LoadItems(
+            PackageItemLoader loader, 
```

Comment:
In general passing in an interface is better than an implementation. Does it make sense to extend the `IItemLoader` interface? Or perhaps adding a new interface that inherits `IItemLoader` a la `IPackageItemLoader`?